SimCalorimetry/HGCalSimProducers/src/HGCDigitizerBase.cc

```diff
@@ -95,4 +95,4 @@ void HGCDigitizerBase<DFr>::updateOutput(std::unique_ptr<HGCDigitizerBase::DColl
 // cause the compiler to generate the appropriate code
 #include "DataFormats/HGCDigi/interface/HGCDigiCollections.h"
 template class HGCDigitizerBase<HGCEEDataFrame>;
-template class HGCDigitizerBase<HGCHEDataFrame>;
+//template class HGCDigitizerBase<HGCHEDataFrame>;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant